src/VersionChip/VersionChip.stories.tsx

```diff
@@ -26,6 +26,15 @@ export const Default = {
   render: Template
 };
 
+export const WholeNumber = {
```

Comment:
I would am more of a fan of the vocabulary of `single digit` vs `whole number` in our current context (we allow 1 and 1.1) the concept of a whole number makes sense, but if the regex is extended in the future to (1.1.1) this is not a mathematically correct number anymore and the concept of a whole number is not really relevant anymore here.

```suggestion
export const SingleDigit= {
```